### PR TITLE
Improve action page ui

### PR DESCRIPTION
**Pull request by blesstosam**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #
WX20190507-164708@2x

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: src/shared/block/block-detail.tsx [hunk 1]
```diff
@@ -283,7 +290,30 @@ export function renderValue(text: string, record: any): JSX.Element | string {
     case "timestamp":
       return <span>{translateFn(record.value)}</span>;
     case "actHash":
-      return <FlexLink path={`/action/${text}`} text={text} />;
+      const content = (
+        <div style={{ display: "flex", justifyContent: "space-around" }}>
```
Comment (CoderZhi) [inline src/shared/block/block-detail.tsx]: According to design, the icon and the label are supposed to center aligned.

Comment (puncsky) [reply src/shared/block/block-detail.tsx]: +1

File: src/shared/block/block-detail.tsx [hunk 2]
```diff
@@ -283,7 +290,30 @@ export function renderValue(text: string, record: any): JSX.Element | string {
...
+          <div>
+            <Icon type="link" style={{ fontSize: "25px" }} />
+            <br />
+            <span style={{ fontSize: "14px" }}>Copy Link</span>
```
Comment (puncsky) [inline src/shared/block/block-detail.tsx]: missing translation

File: src/shared/block/block-detail.tsx [hunk 3]
```diff
@@ -283,7 +290,30 @@ export function renderValue(text: string, record: any): JSX.Element | string {
...
+          </div>
+          <div>
+            <Icon component={EmailSvg} />
+            <br />
+            <span style={{ fontSize: "14px" }}>Email</span>
```
Comment (puncsky) [inline src/shared/block/block-detail.tsx]: missing translation

File: src/shared/block/block-detail.tsx [hunk 4]
```diff
@@ -383,6 +389,41 @@ export function renderValue(text: string, record: any): JSX.Element | string {
...
+            <Icon component={EmailSvg} />
+          </div>
+          <span style={{ fontSize: "12px" }}>{t("action.email")}</span>
+        </a>
+      </div>
+    </div>
+  );
+  return (
+    <span>
+      <FlexLink path={`/action/${text}`} text={text} />
```
Comment (puncsky) [inline src/shared/block/block-detail.tsx]: Screen Shot 2019-05-03 at 2 38 45 PM

need hover cursor for the share button, right?

Comment (puncsky) [reply src/shared/block/block-detail.tsx]: just use antd email icon

Comment (blesstosam) [reply src/shared/block/block-detail.tsx]: there is no email icon in antd design
WX20190504-120525@2x

Comment (uu-z) [reply src/shared/block/block-detail.tsx]: @blesstosam The name of the email button in antd is mail. you can just use that

Comment (puncsky) [reply src/shared/block/block-detail.tsx]: +1

**Pull request merged**